x509/build/build.rs

```diff
@@ -57,14 +57,14 @@ fn gen_init_devid_csr(out_dir: &str) {
         .add_basic_constraints_ext(true, 5)
         .add_key_usage_ext(usage)
         .add_ueid_ext(&[0xFF; 17]);
-    let template = bldr.tbs_template("Caliptra 2.0 Ecc384 IDevID");
+    let template = bldr.tbs_template("Caliptra 2.1 Ecc384 IDevID");
```

Comment:
nit: Could we make constants for these strings because a bunch of them are duplicated?